ql/src/java/org/apache/hadoop/hive/ql/parse/repl/load/message/TableHandler.java

```diff
@@ -91,6 +91,8 @@
       }
 
       return importTasks;
+    } catch (RuntimeException e){
+      throw e;
```

Comment:
Should RuntimeException be added to the "throws" list of this method? Usually compiler catches these errors, so RuntimeException is somehow covered by throws list or it's not expected to be listed explicitly.

Reply:
runtimeException is Runtime and is not required to be declared in the function signature